client/my-sites/plans-features-main/index.jsx

```diff
@@ -379,6 +405,25 @@ export class PlansFeaturesMain extends Component {
 		return false;
 	}
 
+	renderProductsSelector() {
+		if ( ! isEnabled( 'plans/jetpack-backup' ) ) {
+			return null;
+		}
+
+		const { intervalType } = this.props;
+
+		// @todo: Add translations in FormattedHeader once the final copy is provided.
+		return (
+			<div className="plans-features-main__group plans-features-main__group--narrow">
```

Comment:
I wonder if we could use a `is-narrow` classname as a modifier, that's what we usually do around here.

Reply:
Sounds good, I'll update the classname.